pages/api/eventType.ts

```diff
@@ -109,7 +110,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   }, {} as Record<number, EventTypeGroup["eventTypes"][number]>);
   const mergedEventTypes = Object.values(eventTypesHashMap).map((et, index) => ({
     ...et,
-    $disabled: user.plan === "FREE" && index > 0,
+    $disabled: user?.plan === "FREE" && index > 0,
```

Comment:
worth making FREE a constant? or plan is typed in any case then i'm fine with this